README1.md

```diff
@@ -0,0 +1,48 @@
+### Introduction
+The Haversine formulae is one among the most important and useful formula. 
+It is used to calculate the shortest distance between two points on a sphere using their latitudes and longitudes measured along the surface. This way is used in tracking the various objects on earth and majorly used for navigation. The haversine can be expressed in trignometric function as:
+
+                    haversine(θ)=sin^2(θ/2)
```

Comment:
Remove unnecessary indentation. You can style using markdown instead. Read the markdown guide here: https://guides.github.com/features/mastering-markdown/